<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>covertbase</class>
 <widget class="QMainWindow" name="covertbase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模型工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/modeltools.png</normaloff>:/res/modeltools.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="m_sourceList"/>
    </item>
    <item>
     <widget class="QListWidget" name="m_outputList"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionSearch"/>
   <addaction name="actionXMLOut"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>搜索</string>
    </property>
    <addaction name="actionSearch"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="convertini.qrc">
     <normaloff>:/res/open.png</normaloff>:/res/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/res/quit.png</normaloff>
     <normalon>:/res/exit.png</normalon>:/res/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="convertini.qrc">
     <normaloff>:/res/search.png</normaloff>:/res/search.png</iconset>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionCIMIn">
   <property name="icon">
    <iconset>
     <normaloff>:/res/import_cim.png</normaloff>:/res/import_cim.png</iconset>
   </property>
   <property name="text">
    <string>CIM文件导入</string>
   </property>
  </action>
  <action name="actionXMLOut">
   <property name="icon">
    <iconset resource="convertini.qrc">
     <normaloff>:/res/export.png</normaloff>:/res/export.png</iconset>
   </property>
   <property name="text">
    <string>ini转换xxml</string>
   </property>
   <property name="iconText">
    <string>ini转换xml</string>
   </property>
  </action>
  <action name="actionSvg">
   <property name="icon">
    <iconset>
     <normaloff>:/res/svg2.png</normaloff>:/res/svg2.png</iconset>
   </property>
   <property name="text">
    <string>SVG/G文件导入</string>
   </property>
   <property name="toolTip">
    <string>SVG文件转换</string>
   </property>
  </action>
  <action name="actionEIn">
   <property name="icon">
    <iconset>
     <normaloff>:/res/import_E.png</normaloff>:/res/import_E.png</iconset>
   </property>
   <property name="text">
    <string>E文件导入</string>
   </property>
  </action>
  <action name="actionEOut">
   <property name="icon">
    <iconset>
     <normaloff>:/res/export_E.png</normaloff>:/res/export_E.png</iconset>
   </property>
   <property name="text">
    <string>E文件导出</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>配置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="convertini.qrc"/>
 </resources>
 <connections/>
</ui>
